build/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.201-bionic AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-bionic AS build
```

Comment:
Originally the entire sdk tag was used as we were seeing build errors when it differed from the Global.json. If the docker image moves forward before our global.json we may see random breaks in the future